CHANGELOG.md

```diff
@@ -5,15 +5,17 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/)
 
+## 20240816
+- ecs version: 1.86.1
+- al2 ami version: 20240816
```

Comment:
Do we need to mention source AMI versions? Or no since those are not changing?

Reply:
Source AMI is not changing here, so we dont need to mention it.